src/util/strencodings.h

```diff
@@ -115,6 +118,21 @@ T LocaleIndependentAtoi(const std::string& str)
     return result;
 }
 
+/**
+ * LocaleIndependentAtoi64 is provided for backwards compatibility reasons.
+ * Since its behavior differs from standard atoi functionality, it is not a
```

Comment:
In commit "Restore atoi64 compatibility with old versions of Bitcoin Core" (ac1a5b113ccdace4d7f04e2e8c4089e397b5c0b4)

I can't figure how this function is not a valid implementation of the standard function https://en.cppreference.com/w/cpp/string/byte/atoi. If its behavior is actually nonstandard, it would be good to say how.